### PR TITLE
Feat1 Hey Peter, could you check this out

### PR DESCRIPTION
**Pull request by dooshbag14**
pietro line was edited a bit

### CODE REVIEW

**Comment by dooshbag13** [comment]
well i will think about it ok

File: first [hunk 1]
```diff
@@ -20,7 +20,10 @@ sixth line
 SEVEN!!!!! some cont
 <<<<<<< HEAD
```
Comment (dooshbag13) [inline first]: some comments go here

File: first [hunk 2]
```diff
@@ -20,7 +20,10 @@ sixth line
...
 
-PIETROOOOOOOOOOOOO1111111111111111111111111111==GGGGGGGGGGGG1
+
+PIETROOOOOO BADASS OO777777777==111111==7777777777777777
+hahaha1==GGGGGGGGGGGG1
+
```
Comment (dooshbag14) [inline first]: what about this line here

File: first (hunk 1 above)
Comment (dooshbag14) [reply first]: ok got it

File: first (hunk 2 above)
Comment (dooshbag13) [reply first]: its fine bro

**Pull request merged**